{{cookiecutter.project_slug}}/Makefile

```diff
@@ -1,60 +1,57 @@
-CODE_PATH?={{cookiecutter.code_directory}}
+##### PATHS #####
```

Comment:
minor: If you have time, please update same variables `MK_...` in test file `tests/e2e/configuration.py` to preserve consistency (or create an issue, I'll do it later).

Reply:
I will do this in a following-up PR